decidim-core/config/locales/es.yml

```diff
@@ -202,11 +202,16 @@ es:
     newsletter_mailer:
       newsletter:
         note: Has recibido este mensaje porque estás suscrito a boletines de noticias en %{organization_name}. Puedes cambiar la configuración en tu <a href="%{link}">página de notificaciones</a>.
+        see_on_website: ¿No puedes ver este correo electrónico correctamente? Ver en el <a href="%{link}" target="_blank"> sitio web </a>.
```

Comment:
Please, only add translations for English locales. We handle translations via Crowdin, and if we add trannslations directly through the code then the integration gets weird.